docs/modules/announcement_announcement.md

```diff
@@ -0,0 +1,46 @@
+[@LibertyDSNP/sdk](../README.md) / [Exports](../modules.md) / announcement/announcement
+
+# Module: announcement/announcement
+
+## Table of contents
+
+### Functions
+
+- [batch](announcement_announcement.md#batch)
+- [getContract](announcement_announcement.md#getcontract)
+
+## Functions
+
+### batch
+
+▸ `Const`**batch**(`provider`: *Web3*, `accountAddress`: *string*, `uri`: *string*, `hash`: *string*): *Promise*<any\>
+
+batch() allows users call the batch smart contract and post the URI and hash
+of a generated batch to the blockchain.
+
+#### Parameters:
+
+Name | Type | Description |
+:------ | :------ | :------ |
+`provider` | *any* | The web3 instance used for calling the smart contract   |
+`account` | *any* | The account from which to post the batch   |
+`uri` | *string* | The URI of the hosted batch to post   |
+`hash` | *string* | A hash of the batch contents for use in verification   |
+
+**Returns:** *Promise*<any\>
+
+A [web3 contract receipt promise](https://web3js.readthedocs.io/en/v1.3.4/web3-eth-contract.html#id36)
+
+___
+
+### getContract
+
+▸ `Const`**getContract**(`web3Instance`: *any*): *any*
```

Comment:
This is out of date.
I am the reason why we are merging these docs, but now that we are in a public repo, I think we should likely just setup a github pages for them and include the generation in the build step for packaging. I'll add some stories/notes for that.